simulators/ethereum/eth/eth.go

```diff
@@ -14,7 +14,7 @@ func main() {
 	host := hive.New()
 
 	//	get clients
-	// for loop that runs tests against each client type
+	// loop that runs tests against each client type
```

Comment:
Please revert this. I have a commit where I removed this comment completely, and it will just cause a conflict.